specs/DragStarting.md

```diff
@@ -0,0 +1,196 @@
+DragStarting
+===
+
+# Background
+The WebView2 team has been asked to provide a way to override the default drag drop behavior when running in visual hosting mode. This event allows you to know when a drag is initiated in WebView2 and provides the state necessary to override the default WebView2 drag operation with your own logic.
+
+# Examples
+## DragStarting
+Users can use `add_DragStarting` on the CompositionController to add an event handler
+that is invoked when drag is starting. They can use the the event args to start their own
+drag. Notably the `Deferral` can be used to execute any async drag logic and call back into
+the WebView at a later time. The `Handled` property lets the WebView2 know whether to
+exercise its own drag logic or not.
+
+```C++
```

Comment:
We usually also have a sample for C# usage and in this case would be good since it seems like you're using existing WinRT types like Windows.ApplicationModel.DataTransfer.DataPackageOperation .

Reply:
We are no longer projecting to WinRT with initial release. Resolving this